extensions/mongoose/src/__tests__/integration/schemas/Event.schema.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/typeorm
```

Comment:
Please use `lb4 copyright` to update file headers.

Reply:
Done!